run.py

```diff
@@ -33,13 +33,17 @@ def run(args):
     }
     mavros_params = template(f"{dragonfly_dir}/templates/mavros.launch.yaml.template", mavros_parameters)
 
+    runLog = open(f"{dragonfly_dir}/logs/run.log", "w")
```

Comment:
This is not a valid folder in the docker container so the result is no ros logs and no output being directed so no logs at all